zipline/finance/performance/position_tracker.py

```diff
@@ -121,16 +121,15 @@ def calc_gross_value(long_value, short_value):
     return long_value + abs(short_value)
 
 
-def calc_position_stats(pt):
+def calc_position_stats(positions,
```

Comment:
should we just move the body of this into the method or are we ever calling this standalone?